pages/index.js

```diff
@@ -1,30 +1,73 @@
-import React from "react";
+import React, { useRef, useState} from 'react';
 
 import TodoInput from "../components/todo-input";
 import TodoList from "../components/todo-list";
 
 import styled from "styled-components";
 
 export default function Home() {
+
+  const [todos, setTodos] = useState([]);
+  const [index, setIndex] = useState(1);
+
+  const addTodo = ({content, date}) => {
+    if ((content === undefined) || (date === undefined)) {
+      alert('모든 항목을 입력해주세요!');
+      return;
+    }
+
+    else if (String(date).length != 8){
+      alert('날짜를 올바른 형식으로 입력해주세요!');
+      return;
+    }
+
+    const todo = {
+      id: index,
+      content,
+      date,
+      checked: false
+    }
+    setTodos(todos.concat(todo)); 
+    setIndex(index + 1);
+    alert('입력 완료!')
+  }
+
+  const todoRemove = (id) => {
+    setTodos(todos.filter(todo => todo.id !== id))
+  }
+
   return (
     <Wrapper>
+      <Header>리액트-투두</Header>
       <Contents>
-        <TodoInput />
-        <TodoList />
+        <TodoInput addTodo = {addTodo}/>
+        <Space/>
+        <TodoList todos={todos} todoRemove = {todoRemove}/>
```

Comment:
```suggestion
        <TodoInput onSubmit={handleAdd}/>
        <Space/>
        <TodoList {...{todos}} onDelete={handleDelete}/>
```
1. props이름과 전달해줄 값을 담을 변수의 이름이 같을 때에는 spread operator를 이용해 보내주면 좋습니다.
2. 이벤트 핸들러를 정의할 때에는 이름에 접두사 ```handle```을 붙여주고, 자식에게 핸들러를 전달하기 위해 props 이름을 정의할 때에는 접두사 ```on```을 붙여줍니다. [참고](https://blog.sonim1.com/220)
위와 같이 코드 수정해주시면 됩니다.